front/src/styles/scss/applications/operationalStudies/_scenario.scss

```diff
@@ -66,79 +122,110 @@
       height: 100%;
     }
   }
+  .scenario-sidemenu.col-hdp-3.col-xl-4.col-lg-5.col-md-6 {
+    padding: 0 !important;
+  }
 
   .scenario-sidemenu {
     display: flex;
     flex-direction: column;
     height: var(--content-height);
-    color: var(--gray);
-    padding-top: 0.5rem;
-    padding-bottom: 0.5rem;
-    padding: 0 0.5rem;
-    .tabs-container {
-      overflow: initial;
+    background-color: var(--black5);
+    min-width: 24.5rem;
+    max-width: 29.5rem;
+    padding: 0.875rem 1.5rem 0 1.5rem;
+    box-shadow: inset -1px 0 var(--black5);
```

Comment:
```suggestion
    box-shadow: inset -1px 0 var(--black10);
```

Reply:
The parent already have a `var(--black5)`

Reply:
As the background behind already has a black5, we need to keep a black5 to match the mockup color